README.md

```diff
@@ -14,61 +14,148 @@ At Pensano Developers, we are committed to giving back to our community and supp
 We work in agile environments, test-driven development, object-oriented programming, and quickly adapting to new tech stacks. We aim to deliver software that meets the needs of the customer, is flexible and adaptable to change, and is delivered efficiently and with high quality.
 
 # Getting started
+
 1. Clone the repo
 2. Install Node.js dependencies for both front and backend directories
 
-  ```bash
-  cd backend
-  npm install
-  cd ../frontend
-  npm install
-  ```
-3. Install the "react-allergens" package using the package manager of your choice, such as npm or yarn.
-
-To install "react-allergens" using npm:
-  ```bash
-  npm install react-allergens
-  ```
+```bash
+cd backend
+npm install
+cd ../frontend
+npm install
+```
+
+3. Ensure nodemon is installed by running
```

Comment:
Is there a known issue here with nodemon? I did note that nodemon is listed as a dependency where I believe it should be a dev dependency which could potentially be an issue.

Reply:
Hi, yesterday we noticed nodemon appearing in a few error messages. We might need to update it to be a dev dependency then.